hack/set-pipeline-vars.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+FLY_BIN=${FLY_BIN:-fly}
+PIPELINE_NAME="psn-vpc"
+CLUSTER_NAME="sandbox"
+
+$FLY_BIN -t cd-gsp sync
+
+$FLY_BIN -t cd-gsp set-pipeline -p "${PIPELINE_NAME}" \
+	--config "ci/psn.yaml" \
+  --var "account-name=sandbox" \
```

Comment:
This and `CLUSTER_NAME` on line 7 are hardcoded to `sandbox`. We should remember to change them when this is going into the Verify cluster.